src/sentry/lang/javascript/processor.py

```diff
@@ -686,6 +686,7 @@ def process_frame(self, processable_frame, processing_task):
                 if original_function_name is not None:
                     new_frame["function"] = original_function_name
 
+                WEBPACK_WITH_LIBRARY_PREFIX_RE = re.compile(r"webpack:\/\/[\w\-_.]+\/([\w\-_.\/]+)")
```

Comment:
Can we move this regexp compilation webpack branch?

Reply:
Moved the regex to the top, let me know if you meant for anything else